specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -279,6 +279,223 @@ interface DevBoxes {
+        The AAD object id of the user. If value is 'me', the identity is taken from the 
+        authentication context.
+      """)
+      @maxLength(36)
+      @minLength(2)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$|^me$")
+      @path
+      userId: string;
+
+      @doc("The name of a Dev Box.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      devBoxName: string;
+
+      @doc("A customization group name.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      customizationGroupName: string;
+
+      @doc("A customization task ID.")
+      @maxLength(36)
+      @minLength(36)
+      @pattern("^[a-zA-Z0-9]{8}-([a-zA-Z0-9]{4}-){3}[a-zA-Z0-9]{12}$")
+      @path
+      customizationTaskId: string;
+
+      @doc("Accept header.")
+      @header
+      accept: "text/plain";
+    },
+    {
+      @statusCode
+      statusCode: 200;
+
+      @body
+      body: string;
+    }
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Standard operations produce route with /catalogs, but we only allow list on /projects"
+  @doc("Lists all customization tasks available to the project.")
+  @route("/projects/{projectName}/customizationTasks")
+  @get
+  @added(APIVersions.v2024_05_01_preview)
+  listCustomizationTaskDefinitionsByProject is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+    },
+    CustomizationTaskDefinitionResult
+  >;
+
+  @doc("Gets a customization task.")
+  @added(APIVersions.v2024_05_01_preview)
+  getCustomizationTaskDefinitions is StandardResourceOperations.ResourceRead<CustomizationTaskDefinition>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "StandardOperations produces route on an specific customizationTask, but we want it on the group"
+  @doc("Validates a list of customization tasks.")
+  @action("validateGroup")
+  @route("/projects/{projectName}/customizationTasks:validateGroup")
+  @pollingOperation(OperationStatuses.get)
+  @added(APIVersions.v2024_05_01_preview)
+  validateCustomizationTasksAction is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("Customization tasks to validate.")
+      @body
+      body: CustomizationTaskList;
+    },
+    {
+      @statusCode
+      statusCode: 202;
+
+      @header("Location")
+      location: string;
+
+      @pollingLocation
+      @header("Operation-Location")
+      operationLocation: string;
+
+      @body
+      @added(APIVersions.v2024_05_01_preview)
```

Comment:
This operation already has the added decorator over it, we should remove it from this property.
```suggestion
```

Reply:
Compiler throws an error without it?

![image](https://github.com/Azure/azure-rest-api-specs/assets/3045208/f3ba676b-bae1-489e-96e0-ff369d8ceee8)